src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
For example, `Libuv v1.9.1` supports:

![image](https://cloud.githubusercontent.com/assets/8291187/23775502/9cf762dc-04ee-11e7-84d6-2a4a9d87cdf6.png)

And all of our runtime packages all stop at `osx.10.10`:

![image](https://cloud.githubusercontent.com/assets/8291187/23775556/e4258468-04ee-11e7-9113-33b02c70650a.png)

So with your current code, all ASP.NET apps will break on Windows if/when there is a `win11`.

And I can imagine a lot of libraries doing the same thing we do in the runtime by stopping at `osx.10.10` instead of purely `osx`. Thus any app running on `osx.10.13` that uses these libraries will be broken.